litsupport/modules/timeit.py

```diff
@@ -42,17 +46,17 @@ def _mutateCommandLine(context, commandline):
                 "Separate stdout/stderr redirection not "
                 + "possible with traditional output"
             )
-        outfile = context.tmpBase + ".out"
+        outfile = os.path.normpath(context.tmpBase + ".out")
         args += ["--append-exitstatus"]
         args += ["--redirect-output", outfile]
     stdin = cmd.stdin
     if stdin is not None:
         if not os.path.isabs(stdin) and workdir is not None:
-            stdin = os.path.join(workdir, stdin)
+            stdin = os.path.join(workdir, os.path.normpath(stdin))
         args += ["--redirect-input", stdin]
         cmd.stdin = None
     else:
-        args += ["--redirect-input", "/dev/null"]
+        args += ["--redirect-input", "/dev/null"] if os.name != 'nt' else []
```

Comment:
Could you just write this using the if statement syntax instead of using an if expression. i.e.

```
if os.name != 'nt':
    args += ["--redirect-input", "/dev/null"]
```

Reply:
likewise.